### PR TITLE
Telos Integration

### PR DESCRIPTION
**Pull request by kylanhurt**
Refactor EOS activation process and fix activation bugs

Debounce checkHandleAvailability

Add debounce utility function

Adjust EOS activation process

Change EOS plugin method name for API consistency

Fix conflicts and use currencyInfo selector for EOS activation process

Allow extra time for EOS activation async waterfall

Adjust some EOSIO variable and method names per rebase

Point edge-exchange-plugins to version with CoinGecko

Fix some eslint issues in EOSIO activation process

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/actions/CreateWalletActions.js [hunk 1]
```diff
@@ -88,7 +89,7 @@ export const fetchWalletAccountActivationPaymentInfo = (paymentParams: AccountPa
         type: 'WALLET_ACCOUNT_ACTIVATION_ESTIMATE_ERROR',
         data: 'Network Timeout'
       })
-    }, 12000)
+    }, 26000)
```
Comment (kylanhurt) [inline src/actions/CreateWalletActions.js:92]: Debug slow performance issue. Possibly RAM issue?

File: src/components/scenes/CreateWalletAccountSelectScene.js [hunk 2]
```diff
@@ -44,7 +38,8 @@ export type CreateWalletAccountSelectStateProps = {
   isCreatingWallet: boolean,
   paymentDenominationSymbol: string,
   existingCoreWallet: EdgeCurrencyWallet,
-  walletAccountActivationQuoteError: string
+  walletAccountActivationQuoteError: string,
+  currencyConfigs: Object
```
Comment (kylanhurt) [inline src/components/scenes/CreateWalletAccountSelectScene.js:42]: More specific type

File: src/components/scenes/CreateWalletAccountSelectScene.js [hunk 3]
```diff
@@ -6,8 +6,6 @@ import { ActivityIndicator, Image, ScrollView, StyleSheet, View } from 'react-na
 import { Actions } from 'react-native-router-flux'
 import { sprintf } from 'sprintf-js'
 
-import eosLogo from '../../assets/images/currencies/fa_logo_eos.png'
-import steemLogo from '../../assets/images/currencies/fa_logo_steem.png'
```
Comment (kylanhurt) [inline src/components/scenes/CreateWalletAccountSelectScene.js:10]: Are these logos used? If not then delete.

File: src/components/scenes/CreateWalletAccountSetupScene.js [hunk 4]
```diff
@@ -40,7 +34,8 @@ export type CreateWalletAccountSetupOwnProps = {
 
 export type CreateWalletAccountSetupStateProps = {
   handleAvailableStatus: HandleAvailableStatus,
-  isCheckingHandleAvailability: boolean
+  isCheckingHandleAvailability: boolean,
+  currencyConfigs: Object
```
Comment (kylanhurt) [inline src/components/scenes/CreateWalletAccountSetupScene.js:38]: More specific type

File: src/connectors/scenes/CreateWalletAccountSetupConnector.js [hunk 5]
```diff
@@ -6,10 +6,13 @@ import { checkHandleAvailability } from '../../actions/CreateWalletActions.js'
 import { CreateWalletAccountSetup } from '../../components/scenes/CreateWalletAccountSetupScene'
 import type { Dispatch, State } from '../../types/reduxTypes.js'
 
-const mapStateToProps = (state: State) => ({
-  isCheckingHandleAvailability: state.ui.scenes.createWallet.isCheckingHandleAvailability,
-  handleAvailableStatus: state.ui.scenes.createWallet.handleAvailableStatus
-})
+const mapStateToProps = (state: State) => {
```
Comment (kylanhurt) [inline src/connectors/scenes/CreateWalletAccountSetupConnector.js:9]: import dispatch and state props from scene file

**Comment by peachbits** [comment]
Closed in favor of https://github.com/EdgeApp/edge-react-gui/pull/2179

**Pull request closed without merging**